input/index.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css?family=Questrial');
+
+@import "./utility/utility-colors.css";
+@import "./utility/utility-flex.css";
+@import "./utility/utility-list.css";
+@import "./utility/scrollbehaviour.css";
+@import "./utility/utility-heading.css";
+@import "./utility/utility-para.css";
+@import "./utility/utility-fontweight.css";
+@import "./utility/utility-tab.css";
+@import "./utility/utility-icon-size.css";
+@import "./utility/utility-grey-text.css";
+@import "./utility/utility-border-radius.css";
+@import "./utility/utility-padding.css";
+@import "./utility/utility-alignment.css";
+@import "./utility/utility-margin.css";
+@import "./utility/comman.css";
+
+/*universal*/
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  /* font-family: 'Roboto', sans-serif; */
+  font-family: 'Montserrat', sans-serif;
+  word-wrap: break-word;
+  scroll-behavior: smooth;
+}
+
+
+
+:root {
+  --icon-primary-color:#594adb ;
+  --icon-secondary-color: #f17c0e;
+    /* --icon-primary-color:#5468ff; darkblue*/
+      /* --icon-primary-color:#990000 ; 
+  --icon-secondary-color: #FFCC00; */ 
+
+  /* --icon-secondary-color : #7264e8; #f17c0e*/
```

Comment:
Remove the commented part from all the files if not needed

Reply:
done